<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Criador de gráficos personalizados</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbrir">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionSalvar_2"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuGr_fico">
    <property name="title">
     <string>Gráfico</string>
    </property>
    <addaction name="actionTipos_de_gr_fico"/>
    <addaction name="actionPlotar_gr_fico"/>
    <addaction name="actionSalvar_gr_fico"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionFiltrar_dados"/>
    <addaction name="actionConcatenar_dados"/>
   </widget>
   <addaction name="menuAbrir"/>
   <addaction name="menuGr_fico"/>
   <addaction name="menuEditar"/>
  </widget>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Carregar dados</string>
   </property>
  </action>
  <action name="actionSalvar_2">
   <property name="text">
    <string>Salvar Dados</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionTipos_de_gr_fico">
   <property name="text">
    <string>Tipos de gráfico</string>
   </property>
  </action>
  <action name="actionConfigurar_modo_de_leitura">
   <property name="text">
    <string>Leitor de dados</string>
   </property>
  </action>
  <action name="actionTemas">
   <property name="text">
    <string>Layout</string>
   </property>
  </action>
  <action name="actionPlotar_gr_fico">
   <property name="text">
    <string>Plotar gráfico</string>
   </property>
  </action>
  <action name="actionSalvar_gr_fico">
   <property name="text">
    <string>Salvar gráfico</string>
   </property>
  </action>
  <action name="actionFiltrar_dados">
   <property name="text">
    <string>Filtrar colunas</string>
   </property>
  </action>
  <action name="actionConcatenar_dados">
   <property name="text">
    <string>Concatenar arquivos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
